### PR TITLE
Minor typo fix: add missing period at the end of some comment sentences

### PR DESCRIPTION
**Pull request by coolljt0725**
add missing period at the end of some comment sentences.

Signed-off-by: Lei Jitang <leijitang@huawei.com>

### CODE REVIEW

**Comment by wking** [comment]
On Sat, Sep 12, 2015 at 08:21:06PM -0700, Lei Jitang wrote:
> add missing period at the end of some comment sentences.

Looks good to me, although a more specific commit summary would be
nice (‘Fix typo’ isn't very specific ;).

File: config.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ type Root struct {
 type Platform struct {
 	// OS is the operating system.
 	OS string `json:"os"`
-	// Arch is the architecture
+	// Arch is the architecture.
```
Comment (vbatts) [inline config.go]: the periods at the end of this sentence are not required. This terse of a comment can be regarded as the same rules as a bullet list.

File: config.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package specs
 
-// Spec is the base configuration for the container.  It specifies platform
+// Spec is the base configuration for the container. It specifies platform
```
Comment (vbatts) [inline config.go]: in contrast, a sentence with more than one sentence _would_ have ending punctuation.

File: config.go [hunk 3]
```diff
@@ -61,7 +61,7 @@ type MountPoint struct {
 type State struct {
 	// Version is the version of the specification that is supported.
 	Version string `json:"version"`
-	// ID is the container ID
+	// ID is the container ID.
```
Comment (vbatts) [inline config.go]: not needed. see above.

**Comment by vbatts** [comment]
besides the doubled-space fix, nothing in this correction looks to be grammatically needed. I'm going to close this for now.

**Comment by coolljt0725** [comment]
@vbatts  Should I send PR to fix the doubled-space?:)

**Pull request closed without merging**